tests/RulesetSerializationKeyGeneratedTest.php

```diff
@@ -5,11 +5,4 @@
 class RulesetSerializationKeyGeneratedTest extends RulesetTest
 {
     protected $ruleset = 'serialisation-tests/key-generated';
-
-    protected $skipSerializingRules = [
-        '0x00 in dictionary key - serialise only' => "PHP can't parse JSON with null bytes in object keys",
-        '0x00 starting an dictionary key - serialise only' => "PHP can't parse JSON with null bytes in object keys",
-        '0x00 in parameterised list key - serialise only' => "PHP can't parse JSON with null bytes in object keys",
-        '0x00 starting a parameterised list key' => "PHP can't parse JSON with null bytes in object keys",
-    ];
```

Comment:
Null bytes are actually supported within object property names (e.g. `a\0a`), just not as the first character (e.g. `\0a`).

The new schema doesn't require preg-hacking the raw JSON anymore, and the data provider method is able to exclude cases that can't be converted to a PHP object, so these skips are no longer required.